### ISSUE TITLE
New assembly: option to add feature with assembly scars

### ISSUE THREAD

**Issue by manulera** [BjornFJohansson/pydna#193]
In the new assembly implementation, it would be good to have a boolean option to select whether features are added to indicate where joins between fragments happened.

**Comment by BjornFJohansson** [comment]
Yes! I have though of a different solution to this, but never got around to it.
The features is a list property of the seq object. Perhaps, the scar features could be saved in another list that could be appended or not to the feature list. That way they are there, but only visible if the user wants them.

**Comment by manulera** [comment]
I personally think deciding in advance (default=False) whether to include them or not in the normal seqfeature list would be simpler downstream than adding a separate property to the Dseqrecord class.

A separate property would require handling when exporting to gb files and when doing further manipulations to the sequence (e.g. a cut).

What do you think?

**Comment by BjornFJohansson** [comment]
Sounds good as well. Go for it!
